src/templates/en/2021/methodology.html

```diff
@@ -0,0 +1,474 @@
+      </p>
+    </section>
+
+    <section>
+        <h4 id="parsel"><a href="#parsel" class="anchor-link">Parsel</a></h4>
+
+        <p>
+            <a hreflang="en" href="https://projects.verou.me/parsel/">Parsel</a> is a CSS selector parser and specificity calculator, originally written by <a href="{{ url_for('chapter', year=2019, lang=lang, chapter='css') }}">2019 CSS</a> chapter lead <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a> and open sourced as a separate library. It is used extensively in all CSS metrics that relate to selectors and specificity.
+        </p>
+    </section>
+
+    <section>
+      <h2 id="process"><a href="#process" class="anchor-link">Analytical process</a></h2>
+
+      <p>
+        The Web Almanac took about a year to plan and execute with the coordination of more than a hundred <a href="{{ url_for('contributors', year=year, lang=lang) }}">contributors</a> from the web community. This section describes why we chose the chapters you see in the Web Almanac, how their metrics were queried, and how they were interpreted.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="planning"><a href="#planning" class="anchor-link">Planning</a></h3>
+
+      <p>
+        The {{ year }} Web Almanac kicked off in <a href="https://twitter.com/rick_viscomi/status/1273135952848977920">April {{ year }}</a> and, simialr to the 2020 edition, was completed during the continued COVID-19 pandemic. This and other events of {{ year }} were an undercurrent throughout the entire production process and put a lot of additional strain on our contributors beyond the stresses of a fast-paced project like this.
+      </p>
+
+      <p>
+        As we <a href="../2019/methodology#brainstorming">stated in inaugral year&#8217;s Methodology</a>:
+      </p>
+
+      <blockquote>
+        One explicit goal for future editions of the Web Almanac is to encourage even more inclusion of underrepresented and heterogeneous voices as authors and peer reviewers.
+      </blockquote>
+
+      <p>
+        To that end, this year we&#8217;ve continued our changes to the way that we seek and select authors:
+      </p>
+
+      <ul>
+        <li>
+          Previous authors were specifically discouraged from writing again to make room for different perspectives.
+        </li>
+        <li>
+          Everyone endorsing {{ year }} authors were asked to be especially conscious not to nominate people who all look or think alike.
+        </li>
+        <li>
```

Comment:
I removed this section:

```html
        <li>
          Many 2019 authors were Google employees and this year we tried to get a greater balance of perspectives from the broader web community. We believe that the voices in the Web Almanac should be a reflection of the community itself, and not skewed towards any specific company to avoid creating echo chambers.
        </li>
```

We still have a bias, but probably no more so than Google's influence on the web anyway! So don't think we need to specifically call it out. WDYT @rviscomi ?

Reply:
LGTM